kpconv_torch/utils/config.py

```diff
@@ -385,3 +399,29 @@ def save(self):
                 text_file.write(f"epoch_steps = {self.epoch_steps:d}\n")
             text_file.write(f"validation_size = {self.validation_size:d}\n")
             text_file.write(f"checkpoint_gap = {self.checkpoint_gap:d}\n")
+
+    def set_infered_file_path(self, path):
+        self.infered_file = path
+    
+    def get_infered_file_path(self):
+        return self.infered_file
+
+    def get_test_save_path(self):
+        if self.infered_file is not None:
+            test_path = join(Path(self.infered_file).parent, "test", str(self.chosen_log_folder).split("/")[-1])
+        else:
+            test_path = join(self.chosen_log_folder, "test")
+        if not exists(test_path):
+            makedirs(test_path)
+        return test_path
+    
+    def get_train_save_path(self):
```

Comment:
Retourne le bon chemin où sauvegarder le modèle entraîné, selon les options utilisées dans la CLI

Reply:
I'm afraid I do not understand the meaning of `output_folder`. What's the difference with `chosen_log_folder`?

I'm quite confused there. If I go back to the CLI module, you add a `output` argument for the training case, which seems to be related to the `chosen_log` argument. I have the feeling that `output` and `chosen_log` do the same job.

Reply:
What I understand of the algorithm principles is that it can : 
- train an already trained model, pointed by the -l (--chosen-log) option and then it restarts from the last iteration and goes on;
- train a new model, written in a folder pointed by the -o (--output-dir) option. 
In one case, it points on an already existing log and on the other it points on a folder in which the log will be created.

Reply:
Ok let's go for the two-parameter solution!